kalitectl.py

```diff
@@ -219,7 +219,7 @@ def get_pid():
     listen_port = getattr(settings, "CHERRYPY_PORT", LISTEN_PORT)
 
     # Timeout is 1 second, we don't want the status command to be slow
-    conn = httplib.HTTPConnection(LISTEN_ADDRESS, listen_port, timeout=1)
+    conn = httplib.HTTPConnection(LISTEN_ADDRESS, listen_port, timeout=3)
```

Comment:
I don't know how the view that serves the PID request (PING_URL below) goes about its things, but it would be worth trying to optimize this view in all cases.... however, having a sequence like this is not terrible:

Status: Starting up
Status: Not responding
Status: Running (OK)

I mean, the not responding status is mainly for humans to make sense of, so they can hopefully bear with the Pi being a bit slow about things :)